src/bundle/Resources/public/js/scripts/admin.notifications.modal.js

```diff
@@ -98,13 +98,9 @@
         modalTitle.dataset.notificationsTotal = `(${notificationsCount})`;
     };
     const updatePendingNotificationsView = (notificationsInfo) => {
-        const pendingNotificationsExist = notificationsInfo.pending;
-        const userName = doc.querySelector('.ez-user-menu__name');
+        const noticeDot = doc.querySelector('.ibexa-header-user-menu__notice-dot');
```

Comment:
Not sure if we're talking about the same thing, but shouldn't it be called badge instead of dot?

Reply:
> Not sure if we're talking about the same thing, but shouldn't it be called badge instead of dot?

solved in private conversation